checks/check_status.py

```diff
@@ -631,8 +632,28 @@ def body_lines(self):
             "Queue Length: %s" % self.queue_length,
             "Flush Count: %s" % self.flush_count,
             "Transactions received: %s" % self.transactions_received,
-            "Transactions flushed: %s" % self.transactions_flushed
+            "Transactions flushed: %s" % self.transactions_flushed,
+            ""
         ]
+
+        if self.proxy_data:
+            lines += [
+                "Proxy",
+                "=====",
+                "",
+                "  Host: %s" % self.proxy_data.get('host'),
+                "  Port: %s" % self.proxy_data.get('port')
+            ]
+            username = self.proxy_data.get('user')
+            if username:
+                hidden = len(username) / 2 if len(username) <= 7 else len(username) - 4
+                self.hidden_username = '*' * hidden + username[hidden:]
+                self.hidden_password = '*' * len(self.proxy_data.get('password'))
```

Comment:
The number of stars representing the password should preferably be constant to avoid a malicious third party to know its size (and people often share screenshots or send their info page via non-secured email).